### ISSUE TITLE
Documentation for fixing up data after an abnormal exit / disk full

### ISSUE THREAD

**Issue by laher** [oklog/oklog#53]
I have recently caused a 'disk full' situation on an ingeststore node, and this left the data in a state where ingeststore would not start cleanly. (This could also be caused by e.g. an untimely `kill -9`). 

I'd like to document (and clarify) the process for managing this. _How would you want to document this kind of procedure? Some kind of operations manual?_

So, this is my process I documented internally, to remove `LOCK` files and invalid `.pending` files. I think it's roughly the right approach, but I'd like to clarify. Note that my internal docs begin with a link to a 'resizing disks' doc.

1. Try restarting oklog service. If it fails with a 'lock files message, run the following:

```
    cd /media/logs/oklog # wherever your oklog service is storing log data
    pgrep oklog # confirm that oklog is NOT still running
    rm data/store/LOCK data/ingest/LOCK
```

2. Try restarting oklog service again. If it fails with 'short record', it seems like a record was only partly written. It seems to me that this is only likely in a '.pending' file.

```
    cd /media/logs/oklog # wherever your oklog service is storing log data
    mv data/ingest/*.pending /tmp/
```

3. Try restarting oklog again. Should be good now.

**Comment by peterbourgon** [comment]
The LOCK files should recover cleanly, since https://github.com/oklog/oklog/pull/49. Can you paste the specific error you encountered with the short write? Rather than writing an operations manual, I'd prefer to annotate those errors to be more actionable by humans.

**Comment by laher** [comment]
Sounds good. The message was simply:

    short record

It was one of these lines: https://github.com/oklog/oklog/blob/v0.2.0/pkg/store/merge.go#L43 or this https://github.com/oklog/oklog/blob/v0.2.0/pkg/store/merge.go#L168 . Looking at master, these lines are now located in `pkg/store/read.go`

I verified by recompling with a `fmt.Printf` statement above both those panics, and that statement showed me the content of the problematic line (just 2 characters, '01'). Unfortunately the filename isn't passed into this part of the code - it would be nice to have an error message showing the offending file's name.

Cheers.

**Comment by peterbourgon** [comment]
Great, I'll come up with something for this anon.

**Comment by peterbourgon** [comment]
If I'm not mistaken, this should be fixed with 0.3.0.
